linchpin/layouts/kstests.yml

```diff
@@ -0,0 +1,15 @@
+---
+inventory_layout:
+  inventory_file: "{{ resource_name }}.inventory"
+  vars:
+    hostname: __IP__
+  hosts:
+    kstest:
+      count: 1
+      host_groups:
+        - kstest
+    kstest-master:
+      count: 1
+      host_groups:
+        - kstest-master
+
```

Comment:
Please look on the ansible files and remove these blank lines. Some have them some not. It would be nice to have this consistent.